src/App.js

```diff
@@ -175,19 +163,10 @@ export const App = () => {
                             ) : (
                               peer.name
                             )}
-                          </Text>
+                          </BigParagraph>
                         )}
-                        <Text>
-                          Address:{" "}
-                          <a
-                            href={`${EXPLORER_URL}/accounts/${address}`}
-                            target="_blank"
-                            rel="noreferrer"
-                          >
-                            {address}
-                          </a>
-                        </Text>
-                        {hashAmount && <Text>Hash Balance: {hashAmount}</Text>}
+                        <AddressLink address={address} />
```

Comment:
Consolidated this into a little helper component.  Might be overkill but it's cool to go straight to explorer from the name search